File: KanbanAngular/src/app/class/task.ts
import { ProjectMember } from "./project-member";

export class Task {
    taskId:any;
    taskName:any;
    taskDescription:any;
    priority:any;
   cardType:any;
    dueDate:any;
    memberName:any;
    
    constructor(){
        this.taskId="";
        this.taskName="";
        this.taskDescription="";
        this.priority="";
        this.cardType="";
        this.dueDate="";
        this.memberName="";

    }
}

File: KanbanBoard/kanbanservice/src/main/java/com/kanbanboardproject/kanbanservice/domain/Task.java
package com.kanbanboardproject.kanbanservice.domain;

import lombok.*;
import org.springframework.data.annotation.Id;

import java.util.Date;


@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class Task {
    private String taskId;
    private String taskName;
    private String taskDescription;
    private String priority;
    private String cardType;
    private Date dueDate;
    private String memberName;


}
